src/utils.js

```diff
@@ -0,0 +1,68 @@
+export function merge() {
```

Comment:
Could we add som js doc headers to the utilities since they won't be mentioned in user docs